substrate/frame/executive/src/tests.rs

```diff
@@ -460,6 +460,11 @@ impl MultiStepMigrator for MockedModeGetter {
 	fn step() -> Weight {
 		Weight::zero()
 	}
+
+	#[cfg(feature = "try-runtime")]
+	fn try_mbms() -> Result<(), sp_runtime::TryRuntimeError> {
```

Comment:
We used this approach a lot in the past, but i think it can cause some difficult to debug issues.  
Maybe we can try this time to use an extension trait, like `trait MultiStepMigratorTryRuntime : MultiStepMigrator ` that has this function, and then requiring that trait instead.  
You think it could work?

Reply:
What kinds of issues are you referring to and what use case is there? 

My initial reaction is I'm not sure about it, because `MultiStepMigratorTryRuntime` would just be `MultiStepMigrator` with the `try-runtime` feature enabled.